.gitignore

```diff
@@ -19,3 +19,35 @@ language.settings.xml
 
 # python venv files
 virtualenv/
+
+# files autogenerated by gnuradio
```

Comment:
these make sense, I would also expect blocklib/digitizers

Reply:
In the long run I would prefer gnuradio/meson/modtool to create them in `./build` for a proper out of tree build. For now I just added everything that git complained about. That means the files in `blocklib/digitizers` where either added to the git by misstake or the autogenerated files had to be manually adjusted and added to git.